test/command.js

```diff
@@ -807,4 +807,33 @@ describe('Command', function () {
     argv.should.have.property('someone').and.equal('world')
     commandCalled.should.be.true
   })
+
+  describe('positional aliases', function () {
+    it('allows an alias to be defined for a required positional argument', function () {
+      var parser = yargs('yo')
```

Comment:
Small nit: change one of these tests to use `.argv` instead of `.parse()`.

For the tests that use `.parse()`, you can remove the args string passed to `yargs()`.